dbms/src/DataStreams/copyData.cpp

```diff
@@ -51,6 +54,74 @@ void copyDataImpl(IBlockInputStream & from, IBlockOutputStream & to, TCancelCall
 
 inline void doNothing(const Block &) {}
 
+
+void copyDataImpl(BlockInputStreams & froms, BlockOutputStreams & tos)
```

Comment:
Make it `static`

Reply:
Thanks, it has been implemented in the new commit.